### DISCUSSION TITLE
use diferent color_thresholds for difernet lines on the same card

### DISCUSSION THREAD

**Discussion by L0bit0** [kalkih/mini-graph-card#931] [Q&A]
Hi

I basically whant to use color thresholds for temperatures (out an into myhouse) . I like to use difrent colors on diferent measures for each line, but shouw on the same card.
I tried witht hs code:

              type: custom:mini-graph-card
              name: Netatmo Temperatura INT (7d)
              line_width: 0.3
              hours_to_show: 168
              group_by: hour
              show:
                state: false
                fill: true
              entities:
                - entity: sensor.netatmo_int_temperature
                  name: INT
                  color_thresholds:
                    - value: 23
                      color: '#CE5374'
                    - value: 22
                      color: '#02c39a'
                - entity: sensor.netatmo_ext_temperature
                  name: EXT
                  color_thresholds:
                    - value: 0
                      color: '#247ba0'
                    - value: -1
                      color: '#acd9ec'

It did not work, but it also do not works:
![image](https://user-images.githubusercontent.com/10359623/222690886-a4297f7d-8867-479f-a5b5-5a1fd07853b1.png)

I am looking for a mix between these ones:
![image](https://user-images.githubusercontent.com/10359623/222691147-81c17078-1731-482f-94a7-3c380babc6c2.png)

![image](https://user-images.githubusercontent.com/10359623/222691260-a2430bbc-ae5d-44ab-801f-f77d69c2b0c4.png)

Is this is even posible? or color_thresholds will afect both graphs on the same card..

thansk!

**Comment by L0bit0** [comment]
Hi  any idea? or at least confirm if this is possible or not?

thansk

**Reply by ildar170975** [reply]
Not possible currently, there is a FR for this feature in Github

**Reply by L0bit0** [reply]
thanks!

**Comment by Calalei** [comment]
Does anyone know any update regarding to this issue ? I met exactly same case but half year later ,  still not possible ?
Thanks !
